### ISSUE TITLE
Animations Issues

### ISSUE THREAD

**Issue by AlcidesJunior123** [suriyun-production/mmorpg-kit-docs#2169]
Hi, I'm dealing with some glitches in mmorpgkit regarding animations. In the animation for the left hand, when I apply animations of the left hand the character stops performing the combo. The animations are also overlapping with the mounts. Another error is in the jump of the Character controller type mount, even with root motion animations and marked in the script as "Use root motion for jump" it continues to jump wrong. It would be great if the accessories could also have animations like the weapons but that, like the requests for left-hand weapons, do not overlap with the mounts.
 As a suggestion too, I really miss a Ragdoll system here, maybe acting when the character receives a hit from a skill.
 I don't know if I'm doing something wrong with the animations, if you can help, I just noticed because I'm using the weapon slot as a wing. Here's a video of the error.

https://youtu.be/cyHqJ8ZuLFk

**Comment by insthync** [comment]
I want to see your left hand animations settings

**Comment by AlcidesJunior123** [comment]
Well, the only changes I made to the animations were to the left hand weapon entity itself. I defined only the Idle animations, and Run Forward and Left hand weapons attack animations to try to solve the issue of attacks stuck on the first hit. Previously, there were only locomotion animations.

![1](https://github.com/suriyun-production/mmorpg-kit-docs/assets/139077085/2d5966f4-965b-4b36-a993-91143cc0168d)
![1](https://github.com/suriyun-production/mmorpg-kit-docs/assets/139077085/dbd9e35c-dbd4-4636-a4db-00682648093c)
